### DISCUSSION TITLE
CIECAM from Q, s, h not possible?

### DISCUSSION THREAD

**Discussion by briend** [colour-science/colour#680] [General]
I was a bit surprised that the ciecam (02 and cam16) specification require JMh or JCh.  Shouldn't it allow (J or Q), (M or C or s), (h or H)?

**Comment by thomasmansencal** [comment]
Hi @briend,

You should indeed be able to reconstruct the needed data for the models to operate using a lot of different correlates combinations. We support JMh and JCh only at the moment because that is what we need to have the models working in the contexts we use them.

I won't have time to mathematically derive the other combinations anytime soon as there are more pressing issues. If you have them in analytical form somewhere or even better want to contribute them, please feel free to do so!

Cheers,

Thomas

**Comment by briend** [comment]
Thank you, it's not a pressing issue at all as I don't think Q and s are very useful for what I'm doing.  I was thinking maybe there was some other technical reason.  I'm switching my code from colorspacious to colour-science, so that I can use CAM16.  Thanks for all your great work!
